packages/next/taskfile-swc.js

```diff
@@ -67,7 +69,9 @@ module.exports = function (task) {
         },
         jsc: {
           loose: true,
-
+          // Do not enable externalHelpers for server-side code
+          // "_is_native_function.mjs" helper is not compatible with edge runtime
+          externalHelpers: false,
```

Comment:
It seems this is also an issue with the client x-ref: https://github.com/vercel/next.js/pull/37829